lib/chef/handler/datadog.rb

```diff
@@ -160,6 +160,9 @@ def endpoints
         extra_endpoints = @config[:extra_endpoints] || []
         extra_endpoints.each do |endpoint|
           url = endpoint[:url] || config_url()
```

Comment:
can be simplified to `url = endpoint[:api_url] || endpoint[:url] || config_url()` since `endpoint[:api_url]` will return `nil` if `:api_url` isn't set